src/Inkluzitron/Modules/ImagesModule.cs

```diff
@@ -24,15 +27,82 @@ namespace Inkluzitron.Modules
     public class ImagesModule : ModuleBase
     {
         private ProfilePictureService ProfilePictureService { get; }
+        private BotDatabaseContext DbContext { get; set; }
+        private DatabaseFactory DatabaseFactory { get; }
 
-        public ImagesModule(ProfilePictureService profilePictureService)
+
+        public ImagesModule(ProfilePictureService profilePictureService, DatabaseFactory databaseFactory)
         {
+            DatabaseFactory = databaseFactory;
             ProfilePictureService = profilePictureService;
 
             if (!Directory.Exists("ImageCache"))
                 Directory.CreateDirectory("ImageCache");
         }
 
+        protected override void BeforeExecute(CommandInfo command)
+        {
+            DbContext = DatabaseFactory.Create();
+            base.BeforeExecute(command);
+        }
+
+        protected override void AfterExecute(CommandInfo command)
+        {
+            DbContext?.Dispose();
+            base.AfterExecute(command);
+        }
+
+        private static readonly double WhipBdsmTreshold = 0.5; // TODO config maybe?
```

Comment:
Hardcoded hodnoty fuj. Kdyz uz tam mas navic stejnou hodnotu, nedava smysl rovnou pouzit uz existujici TraitReportingThreshold?

Reply:
```tHreshold```

Reply:
Dává, ale byl jsem líný tam přidat to nastavení. Ale teď když to celý předělám na službu, tak to určitě tak udělám

Reply:
Od 02d6f60 se to bere z nastavení modulu